pyproject.toml

```diff
@@ -88,7 +88,7 @@ docs = [
     "mkdocs-get-deps>=0.2.0",
     "mkdocstrings>=0.29.1",
     "mkdocstrings-python>=0.29.1",
-    "linkchecker>=10.2.1",
+    "griffe==1.6.0"  # Required by mkdocstrings-python for API documentation generation.
```

Comment:
Unclear why do we need this explicit dependency, also looks like latest `mkdocstrings-python` depends on `griffe>=1.6.2`, which would be incompatible with this? https://pyoven.org/package/mkdocstrings-python